.github/workflows/checks.yaml

```diff
@@ -9,6 +9,11 @@ on:
         required: false
         type: string
         default: ""
+      coverage:
+        required: false
+        type: number
+        default: 14
```

Comment:
14%?

Reply:
@robert-affinidi I have checked it's our max coverage at that moment

Reply:
It is a parameter which can be overridden on the repo. Let's keep it 80 by default.